source/camera/CullingVectorManager.cpp

```diff
@@ -0,0 +1,105 @@
+/*
+ *  Copyright (C) 2011-2015  OpenDungeons Team
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "camera/CullingVectorManager.h"
+
+#include <algorithm>
+
+void CullingVectorManager::sort()
+{
+    std::vector<VectorInt64> ll[4];
+    VectorInt64 cc(0, 0, 0);
+
+    // find the center of points stored in mMyArray
+    for (auto ii : mMyArray)
+        cc = cc + ii;
+    cc = cc / mMyArray.size();
```

Comment:
after a for boucle without braces, it is better to add an empty line to help reading